packages/abi-coder/src/encoding/strategies/getCoderV1.ts

```diff
@@ -98,7 +98,15 @@ export const getCoder: GetCoderFn = (
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   const components = resolvedAbiType.components!;
 
+  if (resolvedAbiType.type.length > 1000) {
```

Comment:
Should this also be `256` given this [change](https://github.com/FuelLabs/fuels-ts/pull/2358/commits/06d34c0d0d33cb78c2a1e48328582a4022b77e01).

Reply:
Good catch